Update glossary.mdx - Replica Definition (#5023)
dfinity/portal: docs/references/glossary.mdx

#### replica

> The **replica** is an instance of the software implementation of protocol components that are

Reviewer: ```suggestion
The **replica** an instance of software containing all the protocol components 
```
